### DISCUSSION TITLE
File System

### DISCUSSION THREAD

**Discussion by ghost** [KDE/ghostwriter#673] [Ideas]
Hello Sir, Maybe you could make a file system like Obsidian.md because it would be so hassle to open, create and save via file manager.

**Comment by Cultura-Difusa** [comment]
Having a simple _sidebar_ option with a filesystem would be incredible.

**Comment by davquar** [comment]
I agree, the biggest inconvenience that I have when using Ghostwiter is that I have to open each file independently. It would be great to have an integrated filesystem sidebar 👍🏼
